jenkins/end_to_end_tests.sh

```diff
@@ -17,7 +17,14 @@ virtualenv venv
 
 pip install -r requirements/base.txt
 
-# Install pages.
+# Install the page objects from the edx-platform repo.
+# Before doing so, we don't need optimizations for lxml,
+# so install it this way which doesn't bother compiling them.
+STATIC_DEPS=true CFLAGS="-O0"  pip install "lxml==3.4.4"
```

Comment:
this is a nice pattern; you should tell other folks about it if they are waiting on installs.